projects/packages/forms/src/blocks/field-slider/edit.js

```diff
@@ -155,13 +172,29 @@ export default function SliderFieldEdit( props ) {
 							onChange={ onChangeStep }
 						/>
 					</HStack>
+					<HStack alignment="top" className="jp-field-slider-inspector-row">
+						<TextControl
+							__next40pxDefaultSize
+							label={ __( 'Min label', 'jetpack-forms' ) }
+							value={ minLabel }
+							onChange={ onChangeMinLabel }
+						/>
+						<TextControl
+							__next40pxDefaultSize
```

Comment:
```suggestion
							__next40pxDefaultSize
							__nextHasNoMarginBottom
```

Reply:
Done - add added to any other relevant components too.